packages/drawer/src/views/GestureHandlerCompat.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
```

Comment:
Maybe this file could be named as `GestureHandlerStub` which would be more appropriate